### PR TITLE
SharedFunctions - Move all to individual files

### PR DESCRIPTION
**Pull request by ctrlbold**
Moved SharedFunctions functions to \internal

### CODE REVIEW

**Comment by ctrlbold** [comment]
How to test

- [ ] Compare functions found in the old SharedFuntions.ps1 to the functions found in the new directory; \internal, ensuring all of them are there
- [ ] Compare that the function names are the same inside the file as the filename itself
- [ ] See that two functions have been moved to public - Test-SqlConnection and Update-dbatools
- Run a couple commands that require a bunch of internal commands such as Copy-SqlDatabase and Sync-SqlLoginPermissions

File: functions/Test-SqlConnection.ps1 [hunk 1]
```diff
@@ -0,0 +1,222 @@
+Function Test-SqlConnectionTest-SqlConnection
```
Comment (dansqldba) [inline functions/Test-SqlConnection.ps1]: This doesn't look right.

**Comment by dansqldba** [comment]
@ctrlbold has asked for this to go through before anything else. Need to see if any other PR's are making changes to ShareFunctions.

Quick look through the code only one things looks suspect.

File: functions/Test-SqlConnection.ps1 (hunk 1 above)
Comment (ctrlbold) [inline functions/Test-SqlConnection.ps1]: Fixed, thanks

**Pull request merged**